index.js

```diff
@@ -11,6 +11,19 @@ const changePrototype = (to, from) => {
 	Object.setPrototypeOf(to, fromPrototype);
 };
 
+const wrappedToString = (withName, fromBody) => `/* Wrapped ${withName}*/\n${fromBody}`;
+
+const changeToString = (to, from, name) => {
+	const withName = name === '' ? '' : `with ${name.trim()}() `;
```

Comment:
We trim `name` because it's not uncommon to have function names with trailing whitespaces.

```js
t.is(() => {}).bind().name, 'bound ')
```